examples/overlay.html

```diff
@@ -20,38 +25,30 @@
     </div>
   </div>
 <script>
-
-// console.log(localStorage.apiKey);
-// if (!localStorage.apiKey) {
-//     var apiKey = prompt("Paste API Key here")
-//     localStorage.apiKey = apiKey;
-// }
+    console.log(localStorage.apiKey);
+    if (!localStorage.apiKey) {
+        var apiKey = prompt("Paste API Key here")
+        localStorage.apiKey = apiKey;
+    }
 </script>
 
-<script src="https://cdnjs.cloudflare.com/ajax/libs/vue/2.0.3/vue.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.16.4/lodash.js"></script>
-<script src="vue-google-maps.js"></script>
-
 <script>
 
 VueGoogleMap.load({
-    key: 'AIzaSyBzlLYISGjL_ovJwAehh6ydhB56fCCpPQw',
-    v: '3.24',
+    key: localStorage['apiKey'],
```

Comment:
I decided to publish my API key for everyone's convenience. Maybe you want to undo this?